code/processes/housekeeping.q

```diff
@@ -74,13 +74,16 @@ kdbzip:{[FILE]
 \d .unix
 
 //-locates files with path, matching string and age
-find:{[path;match;age;agemin]
+find:{[path;match;age;agemin;checkfordirectory]
+	$[0=checkfordirectory;
+		[fileordir::"files";flag::"f"];
+		[fileordir::"directories";flag::"d"]];
 	$[0=agemin;
-	files:.[{.lg.o[`housekeeping;"Searching for: ",x,y];system "/usr/bin/find ", x," -maxdepth 1 -type f -name \"",y,"\" -mtime +",raze string z};(path;match;age);
+	matches:.[{.lg.o[`housekeeping;"Searching for ",fileordir,": ",x,y];system "/usr/bin/find ", x," -maxdepth 1 -type ",flag," -name \"",y,"\" -mtime +",raze string z};(path;match;age);
```

Comment:
this is one of those things I've been meaning to ask for for a while... 

Could you add a function to torq.q called .proc.sys. Given a string input, it will

a) log it e.g. "executing system command: ", cmd
b) run it in an error trap
c) print success or failure 

Then, instead of calling system here, use .proc.sys

Reply:
should this be done in a separate PR and then used to replace all the "system"s throughout torq?

Reply:
I think a separate PR would help keep this PR in focus.